Web/starter_project_barbies/components/about/ProjectCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ProjectCardProps } from '@/types/AboutTypes'
+import Image from 'next/image'
+
+const ProjectCard = ({ project, isEven }: ProjectCardProps) => {
```

Comment:
instead of `isEven`, i would rather take in `shouldReverseFlex` or sth
basically something that describes the UI because the only reason we need the evenness is to check if it should reverse (we should not care about why it should reverse in here)

Reply:
in this case of course all you'd need to do is rename the variable
but i just wanted to explain the concept